### PR TITLE
THX-34938 upload to jcenter

### PR DESCRIPTION
**Pull request by JustinTullgren**
Consolidated dependencies into the `dependencies.gradle` pattern. 

Renamed the gradle project name and directory for the plugin source to align with the package name in bintray.

All gradle plugin configuration vars/properties are in gradle.properties except bintray user and password.

### CODE REVIEW

File: android-gradle-plugin-aspectj/bintray.gradle [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            publicDownloadNumbers = false
+
+            githubRepo = githubRepository //Optional Github repository
+            githubReleaseNotesFile = 'README.md' //Optional Github readme file
+
+            //Optional version descriptor
+            version {
+                name = pluginVersion //Bintray logical version name
+                //desc = //Optional - Version-specific description'
+                //released  = //Optional - Date of the version release. 2 possible values: date in the format of 'yyyy-MM-dd'T'HH:mm:ss.SSSZZ' OR a java.util.Date instance
```
Comment (JustinTullgren) [inline android-gradle-plugin-aspectj/bintray.gradle]: Do we want to put the date of the release?

File: android-gradle-plugin-aspectj/src/main/kotlin/com/archinamon/lang/kotlin/GroovyInteroperability.kt [hunk 2]
```diff
@@ -1,17 +1,17 @@
 /*
- * Copyright 2016 the original author or authors.
```
Comment (JustinTullgren) [inline android-gradle-plugin-aspectj/src/main/kotlin/com/archinamon/lang/kotlin/GroovyInteroperability.kt:2]: need to combine these two lines

File: android-gradle-plugin-aspectj/bintray.gradle (hunk 1 above)
Comment (JustinTullgren) [reply android-gradle-plugin-aspectj/bintray.gradle]: made the call to do it.

**Review by andreipop — APPROVED** [review]

**Review by andreipop — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by JustinTullgren** [comment]
Not going with maven central since jcenter is a super set. If its needed we can add it later.

**Pull request merged**